tests/liquidateBorrow.ts

```diff
@@ -130,7 +130,7 @@ describe('liquidate borrow', () => {
 	});
 
 	it('liquidate', async () => {
-		await setFeedPrice(anchor.workspace.Pyth, 200, solOracle);
```

Comment:
might need to fix up lint above^

Reply:
`yarn prettify --write` doesnt change anything 🤷